base/sparse/linalg.jl

```diff
@@ -532,10 +531,13 @@ function normestinv{T}(A::SparseMatrixCSC{T}, t::Integer = 2)
     end
```

Comment:
Ah, I was not sure whether to remove this or remove the `t = min(t,n)` line, but then decided that changing `t` silently was a bad idea.  I chose to remove that line and keep the check instead -- I also thought of changing the error to be a warning, and adding the `t = min(t,n)` back after the check.

Aside from thinking that changing silently `t` is a bad idea, I am happy with all other options.